packages/nextjs/utils/scaffold-eth/fetchPriceFromUniswap.ts

```diff
@@ -0,0 +1,14 @@
+import { Fetcher, Route, Token, WETH } from "@uniswap/sdk";
+import { Provider } from "@wagmi/core";
+
+export default async function fetchPriceFromUniswap(provider: Provider) {
```

Comment:
can we export this function through `/utils/scaffold-eth/index.ts`? checkout out how we exported `getLocalProvider` from their

Reply:
Done